lib/logstash/outputs/elasticsearch.rb

```diff
@@ -178,6 +178,8 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   #
   # - index: indexes a document (an event from Logstash).
   # - delete: deletes a document by id
+  # - create_unless_exists: creates a document, raises exception if no id is provided
```

Comment:
wording - "raises exception" <-- exceptions are something developers care about. Maybe instead say "will fail if no ID is provided or if the document already exists?"

The important phrase to change, for me, is "raises exception" (dev focused) to be "fails" (user focused)